appveyor.yml

```diff
@@ -1,5 +1,7 @@
 version: "{build}"
 
+image: Visual Studio 2019
```

Comment:
?

Reply:
This gives a different base image that has Go 1.14.2 preinstalled in C:\go, it appears there is always a version of Go installed so if we install it ourselves we need to put it in another location and adjust the PATH and perhaps some other Go environment variables.

https://www.appveyor.com/docs/windows-images-software/#golang